third_party/docfx-doclet-143274/src/test/java/com/microsoft/model/TocContentsTest.java

```diff
@@ -0,0 +1,69 @@
+package com.microsoft.model;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class TocContentsTest {
```

Comment:
Are there new generated files that should be checked?

Reply:
This test just shows that toc contents is structuring the content as intended. For generated files, I updated the previous tests to include the new param and changed the expected outputs. Updates from this to toc will look like [expected-generated-files/toc.yml](https://github.com/googleapis/java-docfx-doclet/blob/dc6aafae1d812de80b3b4ed2b28909e36a9fa3c0/third_party/docfx-doclet-143274/src/test/resources/expected-generated-files/toc.yml)